### ISSUE TITLE
Link hint obscured by image located next to link

### ISSUE THREAD

**Issue by dragon788** [philc/vimium#2522]
On https://www.imtins.com/ the padlock next to the "Secure login" apparently positions itself as the most foreground element, which obscures the link hint for that particular link.

**Comment by mrmr1993** [comment]
Likely fix is to revert b6125a2b13d6927c10f381f51b207445fd3f049b. See #1162

**Comment by gdh1995** [comment]
@mrmr1993 I agree with commit b6125a2b13d6927c10f381f51b207445fd3f049b and dislike the way to make `z-index` so long.

Such an z-index is not a normal/common usage, so I think Vimium **doesn't need to support it fullly**. 

There're many edge cases that we can not handle correctlly only with JavaScript & CSS - e.g.  CSS `contain: strict` works ugly and breaks many layout rules if it's applied to a full-screen container element.

Just like what smblott said in https://github.com/philc/vimium/issues/2504#issuecomment-299689138:
> That's considerably simpler. This is a bit of a corner case, to which we shouldn't really be devoting large amounts of (hard-to-maintain) logic.
